src/common/window/session-storage.js

```diff
@@ -6,34 +6,68 @@
  * @environment Browser script
  */
 import { generateRandomHexString } from '../ids/unique-id'
-import { isBrowserScope } from '../util/global-scope'
 
-export { getCurrentSessionIdOrMakeNew }
-
-const SESS_ID = 'NRBA_SESSION_ID' // prevents potential key collisions in session storage
+const AGENT_PREFIX = 'NRBA/'
```

Comment:
Do we care that the shape of this has changed?

`NRBA_SESSION_ID` to `NRBA/SESSION_ID`

Unclear if this outlined or defined in the docs anywhere in a concrete way.  If yes, we should match, if no, this is a non-issue.

Reply:
Nope, our internal implementation detail